test/System.IdentityModel.Tokens.Jwt.Tests/JwtSecurityTokenHandlerTests.cs

```diff
@@ -2360,6 +2361,99 @@ public static TheoryData<ParametersCheckTheoryData> ParametersCheckTheoryData()
                 }
             };
         }
+
+        [Theory, MemberData(nameof(SecurityTokenDecryptionTheoryData))]
+        public void GetEncryptionKeys(CreateTokenTheoryData theoryData)
+        {
+            var context = TestUtilities.WriteHeader($"{this}.EncryptionKeysCheck", theoryData);
+            try
+            {
+                TestUtilities.AssertFailIfErrors(context);
```

Comment:
Why do we need this line here?

Reply:
We don't need it. I think I forgot to delete it. I removed it now.